From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Sun, 12 Nov 2017 14:03:31 +0800
Subject: avoid shipping duplicate root.hints and icann-ca.pem

---
 etc/etc.mk | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

Index: knot-resolver/etc/etc.mk
===================================================================
--- knot-resolver.orig/etc/etc.mk
+++ knot-resolver/etc/etc.mk
@@ -1,9 +1,9 @@
-etc_SOURCES := icann-ca.pem \
+etc_SOURCES := \
 	config.cluster \
 	config.isp \
 	config.personal \
-	config.splitview \
-	root.hints
+	config.splitview
+
 
 etc-install: $(DESTDIR)$(ETCDIR)
 	$(INSTALL) -m 0644 $(addprefix etc/,$(etc_SOURCES)) $(DESTDIR)$(ETCDIR)
